_posts/2020-10-01-annual-report-part-1.md

```diff
@@ -39,7 +39,7 @@ Since the report is quite elaborative, we are publishing them in 3 parts:
 
-We use welcoming and open channels for community participation. We have a cohort-based Slack channel and a public-facing Gitter channel ([more details](/ols-2#resources)). We strive to avoid jargon and other non-inclusive language that can alienate, and make underrepresented people feel excluded. We use simple and jargon-free English in our communications, website and training resources (see an example from [one of the cohort calls]({% link _ols-1/week04.md %})). On our website, we encourage people to list their pronouns [in their profile](/ols-1/projects-participants/#participants) if they feel comfortable doing so. At the beginning of each cohort call, we remind participants about our [Code of Conduct]({% link code-of-conduct.md %}) and use a roll call where participants are asked to list their names and pronouns so others address them respectfully. We encourage and recognize the quietest voices, and not just those with the most confidence and volume. During the cohort calls, we facilitate breakout discussions and encourage shared note-taking in shared notes. In OLS-2, cohort calls are live-transcribed using [Otter.ai](https://otter.ai), and some breakout rooms are silent note-based, to ensure that people with varying language skills or ability can still get an interactive and full experience. Recordings from these calls are posted on YouTube for the cohort members to catch up if they couldn't attend the call.
+We use welcoming and open channels for community participation. We have a cohort-based Slack channel and a public-facing Gitter channel ([more details](/ols-2#resources)). We strive to avoid jargon and other non-inclusive language that can alienate, and make underrepresented people feel excluded. We use simple and jargon-free English in our communications, website and training resources (see an example from [one of the cohort calls]({% link _ols-1/week04.md %})). On our website, we encourage people to list their pronouns [in their profile](/ols-1/projects-participants/#participants) if they feel comfortable doing so. At the beginning of each cohort call, we remind participants about our [Code of Conduct](https://github.com/open-life-science/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md) and use a roll call where participants are asked to list their names and pronouns so others address them respectfully. We encourage and recognize the quietest voices, and not just those with the most confidence and volume. During the cohort calls, we facilitate breakout discussions and encourage shared note-taking in shared notes. In OLS-2, cohort calls are live-transcribed using [Otter.ai](https://otter.ai), and some breakout rooms are silent note-based, to ensure that people with varying language skills or ability can still get an interactive and full experience. Recordings from these calls are posted on YouTube for the cohort members to catch up if they couldn't attend the call.
```

Comment:
```suggestion
We use welcoming and open channels for community participation. We have a cohort-based Slack channel and a public-facing Gitter channel ([more details](/ols-2#resources)). We strive to avoid jargon and other non-inclusive language that can alienate, and make underrepresented people feel excluded. We use simple and jargon-free English in our communications, website and training resources (see an example from [one of the cohort calls]({% link _ols-1/week04.md %})). On our website, we encourage people to list their pronouns [in their profile](/ols-1/projects-participants/#participants) if they feel comfortable doing so. At the beginning of each cohort call, we remind participants about our [Code of Conduct]({{ site.github.owner_url }}/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md) and use a roll call where participants are asked to list their names and pronouns so others address them respectfully. We encourage and recognize the quietest voices, and not just those with the most confidence and volume. During the cohort calls, we facilitate breakout discussions and encourage shared note-taking in shared notes. In OLS-2, cohort calls are live-transcribed using [Otter.ai](https://otter.ai), and some breakout rooms are silent note-based, to ensure that people with varying language skills or ability can still get an interactive and full experience. Recordings from these calls are posted on YouTube for the cohort members to catch up if they couldn't attend the call.
```